### PR TITLE
Conformance results for v1.22/digitalocean

### PR DESCRIPTION
**Pull request by elijahomolo**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [ ] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [ ] Did you include the product/project logo in SVG, EPS or AI format?  
* [x ] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [ ] If your product/project is open source, did you include the `repo_url`?
* [ x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

File: v1.22/digitalocean/PRODUCT.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+vendor: DigitalOcean
+name: DigitalOcean Kubernetes
+version: v1.21.7-do.0
```
Comment (waynr) [inline v1.22/digitalocean/PRODUCT.yaml:3]: This should be `1.22.7-do.0`.

File: v1.22/digitalocean/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Conformance testing DigitalOcean Kubernetes
+
+### Setup DigitalOcean Kubernetes cluster
+
+The following instructions will help you configure and create a Kubernetes cluster on DigitalOcean:
+
+```
+$ doctl auth init
+
+$ doctl kubernetes cluster create --version 1.21.2-do.0 do-sonobuoy-1212 --node-pool "name=conformance;size=s-4vcpu-8gb;count=3" --wait
```
Comment (waynr) [inline v1.22/digitalocean/README.md:10]: The version shown here is wrong.

File: v1.22/digitalocean/PRODUCT.yaml (hunk 1 above)
Comment (waynr) [reply v1.22/digitalocean/PRODUCT.yaml:3]: (i don't know for sure, but the wrong version listed here could lead CI checks on this PR to fail)

**Review by waynr — CHANGES_REQUESTED** [review]

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes

**Pull request merged**